src/ignitus-Dashboard/ignitus-Dashboard-Header/Components/dashBoardHeader.js

```diff
@@ -40,64 +53,72 @@ class dashBoardHeader extends React.Component {
-            aria-label="Toggle navigation"
-          >
-            <i className="fa fa-bars" />
-          </button>
-          <div className="collapse navbar-collapse" id="navbarNav">
-            <ul className="nav navbar-nav">
-              <li className="nav-item active">
-                <HashLink smooth to="/#">
-                  <span className="nav-link">
-                    Home 
-{' '}
-<span className="sr-only">(current)</span>
-                  </span>
-                </HashLink>
-              </li>
-              <li className="nav-item">
-                <HashLink smooth to="/#what-we-do">
-                  <span className="nav-link">What we provide?</span>
-                </HashLink>
-              </li>
-              <li className="nav-item">
-                <HashLink smooth to="/#contributors">
-                  <span className="nav-link">Contributors</span>
-                </HashLink>
-              </li>
-              <li className="nav-item">
-                <Link to="/aboutus">
-                  <span className="nav-link" href="/aboutus">
-                    About
-                  </span>
-                </Link>
-              </li>
-              <li className="nav-item">
-                <a className="nav-link disabled" href="#">
-                  Welcome 
-{' '}
-{email}
-                </a>
-              </li>
-              <li className="nav-item">
-                <a className="nav-link" href="#" onClick={this.logout}>
-                  Logout
-                </a>
-              </li>
-            </ul>
           </div>
-        </nav>
+
+          <div className="nav-item element home">
+          <HashLink smooth to="/#">
+            <span className="nav-link">
+              Home 
+            </span>
+          </HashLink>
+          </div>
+          
+          <div className="nav-item element internship">
+          <HashLink smooth to="/#">
+            <span className="nav-link">
+              Internships
+            </span>
+          </HashLink>
+          </div>
+          <div>
+            <span></span>
+          </div>
+
+          <div className="nav-item element">
+            <StatusDropdown />
+          </div> 
+          
+          
+          <div className="nav-item element">
+            <span>
+              <input  type="text" className="searchbox" value={this.state.text} onChange={this.handleChange}></input>
+            </span>
+          </div>
+
+          <div className="btn-floating mx-1 link contact-btn element envelop" >
+          <HashLink
+            href="/"
+            target="_blank" rel="noopener noreferrer" >
+            <i className="fa fa-envelope fa-button" />
+          </HashLink>
+          </div>
+
+          <div className="btn-floating mx-1 link contact-btn element bell" >
+          <a
+            href="/"
+            target="_blank" rel="noopener noreferrer" >
+            <i className="fa fa-bell fa-button" />
```

Comment:
Remove fa- fa icons and import SVG. See the latest code.

Reply:
You can use Icons from here, https://github.com/Ignitus/Ignitus-client/tree/master/src/ignitus-Shared/ignitus-DesignSystem/ignitus-Assets/ignitus-Icons, if the icon you want to use, is not there, then create one by downloading `svg` from` font-awesome / material-icon`, and in the end if you still not feel comfortable with the icon components then you can use it for now, but please remove it in the future.